RemoteHealthcare/BikeManager.cs

```diff
@@ -16,20 +16,19 @@ class BikeManager
         private bool reachedThreshold = false;
 
         private RealBike realBike = null;
+        private SimulatorBike simBike = null;
 
-        //class RealBike : IBikeData
-        //{
-        //    public byte[] Data { get; set; }
-        //    public string ServiceName { get; set; }
+        public void StartSim()
+        {
+            this.simBike = new SimulatorBike();
+            this.simBike.StartSim();
+        }
 
-        //    public RealBike(BLESubscriptionValueChangedEventArgs e)
-        //    {
-        //        this.Data = e.Data;
-        //        this.ServiceName = e.ServiceName;
-        //    }
-        //}
-        public BikeManager()
+        public void SimRunStep()
```

Comment:
Zie [Style Guide](https://github.com/Ti-2-1-RH-A1/remote-healthcare/blob/development/.github/STYLE_GUIDELINES.md#methodes) voor fout bij Methode structuur fout (Danwel hoe methode is opgebouwd of commentaar)